src/bosh-softlayer-cpi/action/create_vm.go

```diff
@@ -95,7 +95,11 @@ func (a CreateVMAction) updateCloudProperties(cloudProps *VMCloudProperties) {
 		a.vmCloudProperties.Domain = "softlayer.com"
 	}
 	helper.LengthOfHostName = len(a.vmCloudProperties.VmNamePrefix + "." + a.vmCloudProperties.Domain)
-
+	// A workaround for the issue #129 in bosh-softlayer-cpi
+	if helper.LengthOfHostName == 64 {
+		a.vmCloudProperties.VmNamePrefix = a.vmCloudProperties.VmNamePrefix + "-1"
```

Comment:
I've tried to experiment with both combinations `"**"` and `".."`, seems that they are not allowed by SL, the following error is being returned. 
>The hostname and domain must be alphanumeric strings that may be separated by periods '.'. The only other allowable special character is the dash '-' However the special characters '.' and '-' may not be consecutive. Each alphanumeric string separated by a period is considered a label. Labels must begin and end with an alphanumeric character and must be between 1-63 characters in length. The last label, the TLD (top level domain) must be between 2-24 alphabetic characters. The TLD label and hostname cannot be solely comprised of digits. The domain portion must consist of least one label followed by a period '.' then ending with the TLD label. Combining the hostname, followed by a period '.', followed by the domain gives the FQDN (fully qualified domain name), which may not exceed 253 characters in total length.

I think the characters contradict with FQDN definition. So I leave the trailing characters as it is unless you or Matt have some other options. 

regards, Vadim